src/api/Jalasoft.TeamUp.Projects.API/local.settings.json

```diff
@@ -1,7 +1,11 @@
 {
-    "IsEncrypted": false,
-    "Values": {
-        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
-        "FUNCTIONS_WORKER_RUNTIME": "dotnet"
-    }
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+  },
+  "Host": {
```

Comment:
is this a common agreement with the team? it's intentional? do Resumes does the same CORS setting?

Reply:
In the backend team yes, we had a common agreement